config/locales/en.yml

```diff
@@ -7,7 +7,7 @@ en:
     tag_migration: 'Bulk tag history'
     tag_importer: 'Bulk tag by upload'
   errors:
-    invalid_taxon: There was a problem with your request and we were notified about it. We will fix this issue as soon as possible
+    invalid_taxon: There was a problem with your request. The chosen slug may already be in use. We have been notified of the issue and will fix it as soon as possible.
```

Comment:
I think the message might have been incorrect. Are we actually being notified about this? (Would have been via Airbrake).